server/src/main/java/org/elasticsearch/index/mapper/DocumentMapper.java

```diff
@@ -76,7 +76,7 @@ private void maybeLog(Exception ex) {
         if (logger.isDebugEnabled()) {
             logger.debug("Error while parsing document: " + ex.getMessage(), ex);
         } else if (IntervalThrottler.DOCUMENT_PARSING_FAILURE.accept()) {
-            logger.error("Error while parsing document: " + ex.getMessage(), ex);
+            logger.info("Error while parsing document: " + ex.getMessage(), ex);
```

Comment:
I believe `ex` is going to be ignored since `log.info` doesn't accept exceptions.